docs/Community_IRC.md

```diff
@@ -30,13 +30,19 @@ In order to enter main `#armbian` channels registration with Nickserv is mandato
 Simply click here: [https://discord.com/invite/armbian](https://discord.com/invite/armbian)  
 
 Channels starting with `#armbian-` are relayed between Discord and Libera IRC so it does not matter if you join IRC or Discord as both ends receive your messages. Check `#welcome-and-rules` for more information.
-The main `#armbian` channel and `#armbian-announcements` are relayed between Discord, Libera and OFTC.
+The main `#armbian` channel and `#armbian-announcements` are relayed between Discord, Libera, OFTC and Matrix.
+
+### Matrix (closed beta)
+
+- Server: `matrix.armbian.com`
+- Channels: `#armbian:matrix.armbian.com` and `#armbian-announcements:matrix.armbian.com` are available only
```

Comment:
I have installed a client ... how to get to the directory?

Reply:
Depends on the client. As for Element there is a button to "Explore rooms"
https://x.na4.eu/KOTu3/TUWUgAzA34.jpg

Reply:
![Selection_081](https://github.com/armbian/documentation/assets/3305006/6487a151-63df-4d0f-8cd2-d2a008c5360e)
Users from different servers have to add our server to get its room list.

Reply:
yes, i got it. Perhaps this has to be exposed. I tried to this from a newbie perspective.

Reply:
I don't think there will be much newbies. Those are usually Discord users :grin:

Reply:
ok